File: test/fancywords/TestFancyWordsServer.java
package fancywords;

import java.sql.SQLException;

/**
 * Created by Shrec on 6/10/2017.
 */
public class TestFancyWordsServer {
    public static void main(String args[]) {
        try {
            FancyWordsServer server = FancyWordsServer.getInstanceAndStart();
            System.out.println(server.requestAllEntries());
        } catch (SQLException e) {
            e.printStackTrace();
        }
    }
}

File: src/jvmTest/kotlin/fancywords/TestFancyWordsServer.kt
package fancywords

import java.sql.SQLException

/**
 * Created by Shrec on 6/10/2017.
 */
fun main() {
    try {
        val server = FancyWordsServer.INSTANCE
        println(server.requestAllEntries())
    } catch (e: SQLException) {
        e.printStackTrace()
    }
}
